components/typography/index.tsx

```diff
@@ -40,6 +42,7 @@ Typography.defaultProps = {
 	variant: 'p',
 	weight: 'regular',
 	color: 'black',
+	colorImmutable: false,
```

Comment:
Not sure if this is the correct approach 🤔.

Reply:
Might not be the most suitable approach, but It's a valid approach as any other... :smile: so it really depends on whether there are other options that solve the problem better, and if there are any suggestions, I'm all ears. :ear:

Reply:
I think we can leave it this way for now.